ArduCopter/mode_rtl.cpp

```diff
@@ -33,6 +33,13 @@ void Copter::ModeRTL::restart_without_terrain()
     }
 }
 
+bool Copter::ModeRTL::navigate_home_complete() const
```

Comment:
can we call this function, "reached_home" to be consistent with "reached_wp_destination" of AC_WPNAV.?